### PR TITLE
ci: build binary and run tests on Windows

### PR DESCRIPTION
**Pull request by wtrocki**

### CODE REVIEW

**Comment by wtrocki** [comment]
Enable CI/CD for tracking issues on windows.
https://github.com/redhat-developer/app-services-cli/issues/828

**Comment by wtrocki** [comment]
Problem replicated. It appears to be inside the library we use

**Comment by wtrocki** [comment]
Problem is with embed.
Even on windows path should be forward slash https://pkg.go.dev/embed

CC @rkpattnaik780 @ankithans

**Comment by wtrocki** [comment]
we should add fmt print to strings loaded by embed.
If that works then we have to log issue with lib

**Review by rkpattnaik780 — APPROVED** [review]

File: .github/workflows/gobuild.yml [hunk 1]
```diff
@@ -3,33 +3,23 @@ name: Go Build
 on:
   push:
     branches: [main]
-    paths:
-      - '**.go'
-      - 'go.mod'
```
Comment (craicoverflow) [inline .github/workflows/gobuild.yml:8]: Why was this removed?

Comment (craicoverflow) [reply .github/workflows/gobuild.yml:8]: I'd rather keep filters in, because this build will run on every change when it does not need to - for example, if I change the Makefile then this will change.

Comment (wtrocki) [reply .github/workflows/gobuild.yml:8]: This PR was used for testing. Changes here are experimental and allowing us to replicate/check if we fixed problem.

**Comment by craicoverflow** [comment]
Obsolete.
